addons/crm/tests/test_crm_lead.py

```diff
@@ -23,6 +23,15 @@ def test_crm_lead_unlink(self):
         # Only Sales manager Unlink the Lead so test with Manager's access rights
         self.env.ref('crm.crm_case_4').with_user(self.crm_salemanager).unlink()
 
+    def test_crm_lead_unlink_calendar_event(self):
+        # Document button in event form view is invisible once related lead
+        # is deleted
+        self.env.ref('crm.crm_case_33').unlink()
+        event_id = self.ref('crm.event_crm_case_33')
+        event = self.env['calendar.event'].browse(event_id)
```

Comment:
event = self.env['calendar.event'].browse(self.ref('crm.event_crm_case_33'))
```suggestion
        event = self.env['calendar.event'].browse(self.ref('crm.event_crm_case_33'))
```